source/gui/settingsDialogs.py

```diff
@@ -3510,13 +3510,13 @@ def isValid(self) -> bool:
 				message = pgettext(
 					"remote",
 					# Translators: This message is presented when the user tries to save the settings with the host or key field empty.
-					"Both host and key must be set in the Remote section in order to automatically connect using an existing Remote Access server after NVDA starts.",
+					"Both host and key must be set in the Remote Access section in order to automatically connect using an existing Remote Access server after NVDA starts.",
```

Comment:
I feel like there's no need to repeat Remote Access

```suggestion
					"Both host and key must be set in the Remote Access section in order to automatically connect using an existing server after NVDA starts.",
```